commit 8f2314c08dd0dd708d3cb5cc453895521d5a50b0 (refs/changes/57/3557/1)
Author: John Levon <john.levon@joyent.com>
Date:   2018-03-07T18:27:58+00:00 (1 year, 7 months ago)
    
    OS-6591 VMX support is not checking for PAT load/save support

diff --git a/usr/src/uts/i86pc/io/vmm/intel/vmx.c b/usr/src/uts/i86pc/io/vmm/intel/vmx.c
index 3aa1d473d6..2689766193 100644
--- a/usr/src/uts/i86pc/io/vmm/intel/vmx.c
+++ b/usr/src/uts/i86pc/io/vmm/intel/vmx.c
@@ -130,6 +130,7 @@ __FBSDID("$FreeBSD$");
 
 #define	VM_EXIT_CTLS_ONE_SETTING					\
 	(VM_EXIT_HOST_LMA			|			\
+	VM_EXIT_LOAD_PAT			|			\
 	VM_EXIT_SAVE_EFER			|			\
 	VM_EXIT_LOAD_EFER			|			\
 	VM_EXIT_ACKNOWLEDGE_INTERRUPT)
